KPP/fullchem/CHANGELOG_fullchem.md

```diff
@@ -34,6 +35,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Unbalanced rxn IONO2 = ISALA is now balanced: IONO2 = ISALA + HNO3
 - Unbalanced rxn IONO2 = ISALC is now balanced: IONO2 = ISALC + HNO3
 
+### Changed
+  - Restored sink reactions for HOI, IONO, IONO2
+  - Use `GCARR_ac` for rxns where the Arrhenius `B` parameter is zero
+
+### Removed
+  - IONO2 recycling
+
```

Comment:
Should these lines be removed? I think they're leftover from a merge since they also occur above.